packages/media-utils/README.md

```diff
@@ -0,0 +1,46 @@
+# Media Utils
+
+The media utils package provides a set of artifacts to abstract media functionality that may be useful in situations where there is a need to deal with media uploads or with the media library, e.g., artifacts that extend or implement a block-editor.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/media-utils --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
+
+## Usage
+
+The package exposes an object under the key `components` we have UI components, and under the key `utils` there are util functions available.
+
+### Utils - mediaUpload
+
+Media upload util is a function that allows the invokers to upload files to the WordPress media library.
+As an example, provided that `myFiles` is an array of file objects, `onFileChange` on onFileChange is a function that receives an array of objects containing the description of WordPress media items and `handleFileError` is a function that receives an object describing a possible error, the following code uploads a file to the WordPress media library:
+```js
+wp.mediaUtils.utils.mediaUpload( {
```

Comment:
Those examples are tightly coupled to WordPress. It should be reflected in the description of the package that it won't work outside of WordPress instance.

See here: https://github.com/WordPress/gutenberg/blob/495608246d3ccdc8e3ff3271eee94d51474acb7e/packages/edit-post/README.md#edit-post

> This package is meant to be used only with WordPress core. Feel free to use it in your own project but please keep in mind that it might never get fully documented.